<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>780</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>780</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>780</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>红黑夜法官</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/image/titleicon.jpg</normaloff>:/image/titleicon.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="picturelabel">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>140</y>
      <width>271</width>
      <height>381</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="form">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>170</y>
      <width>391</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> 
 
color: rgb(255, 11, 137);
font: 12pt &quot;楷体&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="periodlabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>24</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> 
color: rgb(255, 12, 97);
font: 75 24pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>村长选举发言阶段</string>
    </property>
   </widget>
   <widget class="QPushButton" name="speechbutton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>660</y>
      <width>75</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/speechbutton.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="Input_textEdit">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>580</y>
      <width>361</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> 
border-image: url(:/image/outputback.jpg);
 
 
color: rgb(255, 85, 0);
font: 12pt &quot;新宋体&quot;;</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>252</width>
      <height>38</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="daylabel1">
       <property name="font">
        <font>
         <family>新宋体</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 255, 255);</string>
       </property>
       <property name="text">
        <string>天数：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="daylabel2">
       <property name="font">
        <font>
         <family>Palatino Linotype</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 0);</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>90</y>
      <width>124</width>
      <height>35</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="playerlabel2">
       <property name="font">
        <font>
         <family>Lato</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 255, 127);</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="playerlabel1">
       <property name="font">
        <font>
         <family>新宋体</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 255, 0);</string>
       </property>
       <property name="text">
        <string>号玩家</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>550</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="timerlabel">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>550</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> 
font: 75 12pt &quot;Agency FB&quot;;
color: rgb(170, 255, 255);</string>
    </property>
    <property name="text">
     <string>剩余时间</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Output_textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>281</width>
      <height>421</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> 

 
color: rgb(81, 38, 85);
font: 14pt &quot;楷体&quot;;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="endtitlelabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>130</y>
      <width>261</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>48</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> 
font: 48pt &quot;楷体&quot;;
color: rgb(226, 62, 255);</string>
    </property>
    <property name="text">
     <string>游戏结束</string>
    </property>
   </widget>
   <widget class="QLabel" name="endtextlabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>240</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="exitbutton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>430</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> 
border-image: url(:/image/exit.png);
border-image: url(:/image/exit.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="restartbutton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>430</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> 
border-image: url(:/image/restart.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="tablelabel">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>130</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>隶书</family>
      <pointsize>28</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
font: 28pt &quot;隶书&quot;;</string>
    </property>
    <property name="text">
     <string>玩家信息表</string>
    </property>
   </widget>
   <widget class="QLabel" name="speechtextlabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>隶书</family>
      <pointsize>28</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 255, 255);
font: 28pt &quot;隶书&quot;;</string>
    </property>
    <property name="text">
     <string>发言内容</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>204</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="daylabel">
       <property name="font">
        <font>
         <family>新宋体</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>当前阶段：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="daylabel_c">
       <property name="font">
        <font>
         <family>新宋体</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>白天</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>picturelabel</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>form</zorder>
   <zorder>periodlabel</zorder>
   <zorder>speechbutton</zorder>
   <zorder>Input_textEdit</zorder>
   <zorder>lcdNumber</zorder>
   <zorder>timerlabel</zorder>
   <zorder>Output_textEdit</zorder>
   <zorder>endtitlelabel</zorder>
   <zorder>endtextlabel</zorder>
   <zorder>exitbutton</zorder>
   <zorder>restartbutton</zorder>
   <zorder>tablelabel</zorder>
   <zorder>speechtextlabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
